game_state.py

```diff
@@ -0,0 +1,122 @@
+from pygame import K_w, K_a, K_s, K_d, K_F11, K_ESCAPE # import wasd controls
+from pygame import K_UP, K_DOWN, K_RIGHT, K_LEFT
+from ScreenHandler import ScreenHandler
+class Universe:
+    """
+    class that should contain all of the players, characters, and should probably own the camera position too?
```

Comment:
Instead of creating a universe class, it would probably be better to have it be a level manager, and rename the world class in World.py to level so its more descriptive of what it actually is since it will contain all the entities and tiles.

Reply:
I like it, I was just throwing something against the wall to see what stuck.